cards/document-standard/component.js

```diff
@@ -48,7 +48,11 @@ class document_standardCardComponent extends BaseCard['document-standard'] {
         eventType: 'CTA_CLICK',
         eventOptions: this.addDefaultEventOptions(),
         // ariaLabel: '',
-      }
+      },
+      feedback: false, // Shows thumbs up/down buttons to provide feedback on the result card
```

Comment:
These are set to false on every card but the buttons are still appearing on every card on the test site

Reply:
I'll add that